### DISCUSSION TITLE
v0.5.7

### DISCUSSION THREAD

**Discussion by clgwlg** [ctripcorp/flybirds#171] [Announcements]
# Features
1. [Android] 支持新版airtest ffmpeg录屏模式, 如需使用，在flybirds_config.json的frame_info中添加配置："airtestRecordMode": "ffmpeg"，支持ffmpeg和yosemite两种模式
2. [web] 支持切换浏览器tab：切换目标页面标题[{title}]链接[{url}]
4. [web] 支持执行js：执行js[{param}]
5. [web] 支持xml request对比：验证服务请求[{service}]与xml路径[{target_data_xml_path}]一致
6. [web] 支持dom元素文本对比：对比目标元素的链接[{target_url}]与文本内容[{target_ele}]和比较元素的链接[{compared_url}]与文本匹配内容[{compared_ele}]
7. [web] 支持图像对比：对比目标图片[{target_picture_path}]和比较图片[{compared_picture_path}]
8. [web] 支持hover操作：悬浮[{selector}]
9. 项目template中增加了对应的例子

# Requirement
1. airtest>=1.2.10

<hr /><em>This discussion was created from the release <a href='https://github.com/ctripcorp/flybirds/releases/tag/v0.5.7'>v0.5.7</a>.</em>

**Comment by wanghaisheng** [comment]
@clgwlg web端经常有那种提交图片、pdf、视频的操作，这个目前doc里我好像没找到

**Reply by clgwlg** [reply]
目前还不支持，我看了下，应该可以做

**Reply by clgwlg** [reply]
提个issue吧
